aiida/utils/cli/options.py

```diff
@@ -44,60 +44,59 @@ def __call__(self, *args, **kwargs):
 
         return click.option(*args_copy, **kw_copy)
 
+
 code = overridable_option(
     '-c', '--code', type=click.STRING, required=True,
     callback=validators.validate_code,
     help='the label of the AiiDA code object to use'
 )
 
+
 structure = overridable_option(
     '-s', '--structure', type=click.INT, required=True,
     callback=validators.validate_structure,
     help='the node pk of the structure'
 )
 
+
 pseudo_family = overridable_option(
     '-p', '--pseudo-family', type=click.STRING, required=True,
     callback=validators.validate_pseudo_family,
     help='the name of the pseudo potential family to use'
 )
 
+
 kpoint_mesh = overridable_option(
     '-k', '--kpoint-mesh', 'kpoints', nargs=3, type=click.INT, default=[2, 2, 2], show_default=True,
     callback=validators.validate_kpoint_mesh,
     help='the number of points in the kpoint mesh along each basis vector'
 )
 
-parent_calc = overridable_option(
-    '-r', '--parent-calc', type=click.INT, required=True,
-    callback=validators.validate_parent_calc,
-    help='the node pk of the parent calculation'
+
+calculation = overridable_option(
+    '-r', '--calculation', type=click.INT, required=True,
```

Comment:
what about using uppercase `-C`?
as it stands, it really isn't apparent why it would be `-r`